portfolio/src/main/java/com/google/sps/data/CommentList.java

```diff
@@ -0,0 +1,40 @@
+
+package com.google.sps.data;
+import java.util.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+//list of comments sorted by date
+public class CommentList{
```

Comment:
This class duplicates a lot of functionality of ArrayList. I think I must be missing something. Why not use ArrayList<Comment> instead of CommentList?